cli/utils/calculate-rewards.ts

```diff
@@ -115,9 +126,10 @@ function updateBalanceByBlock(
   account: string,
   blockNumber: number
 ) {
-  if (state.attestationCompletions[account] >= 3) {
-    const balancesByBlock = state.balancesByBlock[account]
-    if (balancesByBlock.balances.length > 0) {
+  // @ts-ignore
```

Comment:
What's this ts-ignore for? Can you add a comment if it's necessary?

Reply:
that was stale. Thank you